debugger/breakpoints.ml

```diff
@@ -30,10 +31,10 @@ let debug_breakpoints = ref false
 let breakpoint_number = ref 0
 
 (* Breakpoint number -> event. *)
-let breakpoints = ref ([] : (int * debug_event) list)
+let breakpoints = ref ([] : (int * code_event) list)
```

Comment:
Arguably useless, but I would try and get rid of `int`
types that do not represent quantities, by defining
*e.g.* `type breakpoint_id = int`.